code/datums/wounds/_wounds.dm

```diff
@@ -110,6 +110,8 @@
  * * smited- If this is a smite, we don't care about this wound for stat tracking purposes (not yet implemented)
  */
 /datum/wound/proc/apply_wound(obj/item/bodypart/L, silent = FALSE, datum/wound/old_wound = null, smited = FALSE)
+	return
+	/*
 	if(!istype(L) || !L.owner || !(L.body_zone in viable_zones) || !L.is_organic_limb() || HAS_TRAIT(L.owner, TRAIT_NEVER_WOUNDED))
```

Comment:
This is a very lazy way to remove wounds.

Reply:
Sadly, I'm not qualified to remove wounds atm, We just need a hotfix to get this OUT now. It's been a pain in our ass for 6 months and I'm the first person to even try to address it.